.evergreen/cleanup-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Suggest renaming this file to "stop-kmip-server.sh" to mirror "start-kmip-server.sh".